gradle.properties

```diff
@@ -14,7 +14,7 @@
 # limitations under the License.
 #
 
-ideaEdition = IU
+ideaEdition = IC
```

Comment:
Does it make more sense to leave it IU for combined functionality? It feels we have too many features for IU from app engine and they could be essentially ignored unless we keep changing this every time. We probably need to configure CI to build with both IC/UI?

Reply:
yeah. makes sense. will revert